### ISSUE TITLE
Two templates only create a same js file and a same css file

### ISSUE THREAD

**Issue by benjamin-zuo** [zont/gulp-usemin#102]
### index.tpl
```HTML
<!doctype html>
<html>
  <head>
	<meta charset="utf-8">
	<title>Test code deployment</title>
	<!--build:css css/index.css -->
	<link rel="stylesheet" href="../css/common.css">
	<link rel="stylesheet" href="../css/layout.css">
	<!-- endbuild -->
  </head>
  <body>
	<div class="nav">nav</div>


	<script src="../js/jquery.js"></script>
	
	<!-- build:js js/index.js -->
	<script src="../js/common.js"></script>
	<script src="../js/layout.js"></script>
	<!-- endbuild -->
  </body>
</html>
```

### userlist.tpl
```HTML
<!doctype html>
<html>
  <head>
	<meta charset="utf-8">
	<title>Test code deployment</title>
	<!--build:css css/userlist.css -->
	<link rel="stylesheet" href="../css/common.css">
	<link rel="stylesheet" href="../css/navbar.css">
	<!-- endbuild -->
  </head>
  <body>
	<div class="nav">nav</div>


	<script src="../js/jquery.js"></script>
	
	<!-- build:js js/userlist.js -->
	<script src="../js/common.js"></script>
	<script src="../js/navbar.js"></script>
	<!-- endbuild -->
  </body>
</html>
```
```Javascript
gulp.task('usemin', ['clean'], function() {

  return gulp.src('./src/templates/*.tpl')
		.pipe(usemin({
		  css: [minifyCSS(), rev()],
		  js:[uglify(), rev()]
		}))
		.pipe(gulp.dest('./build/'));
})
```
###  index.tpl
```HTML
<!doctype html>
<html>
  <head>
	<meta charset="utf-8">
	<title>Test code deployment</title>
	<link rel="stylesheet" href="css/index-4c6329a1.css"/>
  </head>
  <body>
	<div class="nav">nav</div>


	<script src="../js/jquery.js"></script>
	
	<script src="js/index-74f76f40.js"></script>
  </body>
</html>
```
### userlist.tpl

```HTML
<!doctype html>
<html>
  <head>
	<meta charset="utf-8">
	<title>Test code deployment</title>
	<link rel="stylesheet" href="css/index-4c6329a1.css"/>
  </head>
  <body>
	<div class="nav">nav</div>


	<script src="../js/jquery.js"></script>
	
	<script src="js/index-74f76f40.js"></script>
  </body>
</html>
```

**Comment by caoke90** [comment]
They are included in js/index-74f76f40.js

**Comment by benjamin-zuo** [comment]
I get it  when I change  to  . ~_~

**Comment by zont** [comment]
Fixed in 0.3.14
